lib/partiql-isl/build.gradle

```diff
@@ -0,0 +1,159 @@
+import org.partiql.pig.cmdline.Command
```

Comment:
Should we also add a task to the [GH actions workflow](https://github.com/partiql/partiql-lang-kotlin/blob/96d00f0cacaec80abc880e0ce49077fb6a7cbb48/.github/workflows/build.yml) to test `partiql-isl-kotlin`?

Reply:
I see. Did we also want to move the [tests](https://github.com/partiql/partiql-isl-kotlin/tree/main/test) from the `partiql-isl-repo` to `partiql-lang-kotlin`?

Reply:
Ah that's my miss. Turns out all the sources are in the `src` directory haha

Reply:
Included